addons/sys_rack/stringtable.xml

```diff
@@ -3,27 +3,51 @@
     <Package name="sys_rack">
         <Key ID="STR_ACRE_sys_rack_mountRadio">
             <English>Mount Radio</English>
+            <German>Funkgerät einbauen</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_unmountable">
             <English>Unmountable (Locked)</English>
+            <German>Funkgerät fest verbaut</German>
         </Key>
         <Key ID="STR_ACRE_sys_radio_noGuiExternal">
             <English>Radio used externally. Only owner may configure.</English>
+            <German>Das Funkgerät wird extern benutzt. Nur der Besitzer kann es einstellen.</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_unmountRadio">
             <English>Unmount Radio (%1)</English>
+            <German>Entferne Funkgerät (%1)</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_mountedRadio">
             <English>Mounted radio (%1)</English>
+            <German>Eingebautes Funkgerät (%1)</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_stopUsingRadio">
             <English>Stop using radio</English>
+            <German>Benutzung beenden</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_useRadio">
             <English>Use radio</English>
+            <German>Funkgerät verwenden</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_unableUnmount">
             <English>Unable to unmount radio as you have no inventory space.</English>
+            <German>Funkgerät kann nicht entfernt werden. Nicht genug Platz im Inventar vorhandne.</German>
+        </Key>
+        <Key ID="STR_ACRE_sys_rack_Racks">
+            <English>Racks</English>
+            <German>Funkgeräteschrank</German>
+        </Key>
+        <Key ID="STR_ACRE_sys_rack_Dash">
+            <English>Dash</English>
+            <German>Steckplatz</German>
```

Comment:
The problem i see, is that you are going to mix german an english strings.
That's why I think an translation of this interaction is mandatory.